### ISSUE TITLE
When write the react-router's  Route into the Box tag, the router is not working.

### ISSUE THREAD

**Issue by billyct** [segmentio/ui-box#22]
when i write the code like below, the router is not working.

```
<Box>
<Route path="/a" component={A}/>
<Route path="/b" component={B}/>
</Box>
```

here's an example, i copied the code from react-router's basic example, and change the div with Box.

[![Edit jn8krovrwv](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/jn8krovrwv)

**Comment by mshwery** [comment]
I believe [the latest commit](https://github.com/segmentio/ui-box/commit/756f9f915bb89cfb8c0805200249ad63f9f0ea00#diff-93f88c0d1a613080833a32d5d6dcc885) to master will solve your problem. Previously ui-box used `React.PureComponent` which doesn't rerender appropriately based on react-router-dom's implementation. 

Once that commit is released (cc @Rowno @jeroenransijn) it should work properly for you.

**Comment by Rowno** [comment]
I've released [1.3.0](https://github.com/segmentio/ui-box/releases/tag/v1.3.0) which fixes the bug. Thanks for reporting it. 🙂
